### ISSUE TITLE
2nd Level Contact tracing : Yellow warning code -  Zweite Grades Kontaktnachverfolgung :Gelbe Code

### ISSUE THREAD

**Issue by Vineet-Bhola** [corona-warn-app/cwa-wishlist#431]

**Comment by Ein-Tim** [comment]
Closely related to #24

**Comment by DanAmt** [comment]
I agree that the time to information ( let's call it TTI)need converge to zero. I suggest to make this a goal of the App. Along with your suggest, I want to suggest to have a report of yellow , when someone gets PCR tested or has a positive rapid test, since this gap also adds two to three days.

**Comment by di-sc** [comment]
> I agree that the time to information ( let's call it TTI)need converge to zero. I suggest to make this a goal of the App. Along with your suggest, I want to suggest to have a report of yellow , when someone gets PCR tested or has a positive rapid test, since this gap also adds two to three days.

@DanAmt 
Notifying positive rapid tests is already announced as a new feature of the CWA for second half of april.
Notifying contacts before receiving a test result is not a good idea. In this case you may warn and scare people even if a test result is later negative. Additionally this won't help because delays may occur also **before** having the PCR test, e.g. not getting an appointment for a test in time or not having a test on first symptoms, etc.
I highly recomment to implement the feature of notifying only on a positive test (PCR or rapid test): the primariy contact with a red notification **and** all contacts on >= second level with a yellow/orange notification.
(see #24 and #403 where the feature is discussed in more details)

**Comment by heinezen** [comment]
Hello @Vineet-Bhola ,

Thank you for your contribution. The request you make is very similar to the topics discussed in https://github.com/corona-warn-app/cwa-wishlist/issues/24 . I suggest you follow the discussion there and mirror your comment to the thread as some of the points you raised have already been mentioned there.

I will close your issue as a duplicate of https://github.com/corona-warn-app/cwa-wishlist/issues/24 in order to keep the discussion in one central place.

Regards,
CH

----
Corona-Warn-App Open Source Team
